cmd/tsidp/tsidp.go

```diff
@@ -181,7 +305,7 @@ type authRequest struct {
 	validTill time.Time
 }
 
-func (ar *authRequest) allowRelyingParty(ctx context.Context, remoteAddr string, lc *tailscale.LocalClient) error {
+func (ar *authRequest) allowRelyingParty(ctx context.Context, remoteAddr string, lc *tailscale.LocalClient, clientID string, clientSecret string) error {
```

Comment:
document clientID and clientSecret?

Reply:
// allowRelyingParty validates that the relying party identified by either the remoteAddr or clientID/clientSecret pair can ...